client/src/test/java/org/spine3/protobuf/MessageFieldShould.java

```diff
@@ -20,18 +20,106 @@
 
 package org.spine3.protobuf;
 
+import com.google.protobuf.Message;
+import com.google.protobuf.StringValue;
 import org.junit.Test;
 
+import static com.google.protobuf.Descriptors.FieldDescriptor;
+import static com.google.protobuf.Descriptors.FieldDescriptor.JavaType;
 import static org.junit.Assert.assertEquals;
+import static org.spine3.base.Identifiers.newUuid;
+import static org.spine3.protobuf.Values.newStringValue;
 
 @SuppressWarnings("InstanceMethodNamingConvention")
 public class MessageFieldShould {
 
+    private static final int STR_VALUE_FIELD_INDEX = 0;
+
+    @SuppressWarnings("DuplicateStringLiteralInspection")
+    public static final String STR_VALUE_FIELD_NAME = "value";
+
+    private final StringValue stringValue = newStringValue(newUuid());
+
+    @Test
+    public void have_index() {
+        final int index = 5;
```

Comment:
Do you check for a numeric value to pass? Why do you call it `index` then? It looks like a proto field index, judging by its name.